package.json

```diff
@@ -24,21 +27,37 @@
-    "b:esm": "esbuild --bundle src/opentype.mjs --outdir=dist --target=es2018 --format=esm  --out-extension:.js=.mjs     --define:DEBUG=false",
-    "d:esm": "esbuild --bundle src/opentype.mjs --outdir=dist --target=es2018 --format=esm  --out-extension:.js=.min.mjs --define:DEBUG=false --minify --sourcemap"
+    "start": "node esbuild-runner.mjs --input src/opentype.mjs --output dist/opentype.mjs --externals \"['fs']\" --target es2015 --module --watch --servedir . --global-name opentype --footer \"(function (root, factory) { if (typeof define === 'function' && define.amd)define(factory); else if (typeof module === 'object' && module.exports)module.exports = factory(); else root.opentype = factory(); })(typeof self !== 'undefined' ? self : this, function(){return ((function(a,b){var keys=Object.keys(b);for(var i=0;i<keys.length;i++)a[keys[i]]=b[keys[i]];return a})({'default':opentype},opentype));});\"",
+    "b:umd": "node esbuild-runner.mjs --input src/opentype.mjs --output dist/opentype.js  --externals \"['fs']\" --target es2015 --global-name opentype --footer \"(function (root, factory) { if (typeof define === 'function' && define.amd)define(factory); else if (typeof module === 'object' && module.exports)module.exports = factory(); else root.opentype = factory(); })(typeof self !== 'undefined' ? self : this, function(){return ((function(a,b){var keys=Object.keys(b);for(var i=0;i<keys.length;i++)a[keys[i]]=b[keys[i]];return a})({'default':opentype},opentype));});\"",
+    "d:umd": "node esbuild-runner.mjs --input src/opentype.mjs --output dist/opentype.min.js  --externals \"['fs']\" --target es2015 --minify --sourcemap --global-name opentype --footer \"(function (root, factory) { if (typeof define === 'function' && define.amd)define(factory); else if (typeof module === 'object' && module.exports)module.exports = factory(); else root.opentype = factory(); })(typeof self !== 'undefined' ? self : this, function(){return ((function(a,b){var keys=Object.keys(b);for(var i=0;i<keys.length;i++)a[keys[i]]=b[keys[i]];return a})({'default':opentype},opentype));});\"",
+    "b:esm": "node esbuild-runner.mjs --input src/opentype.mjs --output dist/opentype.mjs --externals \"['fs']\" --target es2015 --module",
+    "d:esm": "node esbuild-runner.mjs --input src/opentype.mjs --output dist/opentype.min.mjs --externals \"['fs']\" --target es2015 --module --minify --sourcemap",
+    "b:compat": "node esbuild-runner.mjs --input src/opentype.mjs --output dist/opentype.compat.js  --externals \"['fs']\" --target compat --global-name opentype --footer \"(function (root, factory) { if (typeof define === 'function' && define.amd)define(factory); else if (typeof module === 'object' && module.exports)module.exports = factory(); else root.opentype = factory(); })(typeof self !== 'undefined' ? self : this, function(){return ((function(a,b){var keys=Object.keys(b);for(var i=0;i<keys.length;i++)a[keys[i]]=b[keys[i]];return a})({'default':opentype},opentype));});\"",
```

Comment:
so `bun`, `deno` et al. cannot be used anymore to build opentype.js ?

Reply:
If I understand correctly, `b:compat` is the only target that require a post processing via SWC. If that's the case, maybe we could keep the other target as they were (ie. not limited to node) and only use the `node esbuild-runner.mjs` for the b:compat target ?

Reply:
> If I understand correctly, `b:compat` is the only target that require a post processing via SWC. If that's the case, maybe we could keep the other target as they were (ie. not limited to node) and only use the `node esbuild-runner.mjs` for the b:compat target ?

The other ones do use processing via SWC to keep them compatable with es2015 even if we include more advanced features from later ECMAScript versions.

Reply:
> so now that we explicit `node` in our command, `bun`, `deno` et al. cannot be used anymore to build opentype.js ?

can't they just do this?:
```bash
alias node deno
npm run build
```

Reply:
understood, so all the target goes through SWC to be compatible with old browser.
in this case, what is the point of b:compat ?